SDK Showcase/SDK Showcase/Interactors/SDKInteractor.swift

```diff
@@ -0,0 +1,29 @@
+//  Copyright © 2025 Onewelcome Mobile Identity. All rights reserved.
+
+import SwiftUI
+import OneginiSDKiOS
+
+//MARK: - Protocol
+protocol SDKInteractor {
+    func initializeSDK(result: @escaping SDKResult)
+}
+
+//MARK: - Stubb methods
+struct StubSDKInteractor: SDKInteractor {
```

Comment:
I would rather suffix with `Stub`, not prefix e.g. how about `SuccessSDKInteractorStub`?